src/main/webapp/index.html

```diff
@@ -3,47 +3,103 @@
   <head>
 <html lang="en">
 <head>
+    <meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests">
     <meta charset="UTF-8">
     <title>Project</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
     <script src="https://accounts.google.com/gsi/client" async defer></script>
     <script src="https://unpkg.com/jwt-decode/build/jwt-decode.js"></script>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SPS Team39</title>
+    <link rel="stylesheet" href="style/style.css">
   </head>
   <body>
-    <div id="login">
-        <div id="buttonDiv"></div> 
-        <p id="fullName"></p>
-        <p id="sub"></p>
-        <p id="given_name"></p>
-        <p id="family_name"></p>
-        <p id="email"></p>
-        <p id="verifiedEmail"></p>
-        <img id="picture" />
-      <meta http-equiv="X-UA-Compatible" content="IE=edge">
-      <meta name="viewport" content="width=device-width, initial-scale=1.0">
-      <title>SPS Team39</title>
-      <link rel="stylesheet" href="style/style.css">
-  
-  </head>
-  <body>
+
       <div class="mainWrapper">
           <div class="welcomeWrapper">
               <h1>Welcome to SPS Team39's Project!</h1>
```

Comment:
Feel free to update this with a webapp name! https://namelix.com/ if you need inspiration.